pkg/kubectl/cmd/create/create_tenant.go

```diff
@@ -32,13 +33,19 @@ var (
 		Create a tenant with the specified name.`))
 
 	tenantExample = templates.Examples(i18n.T(`
-	  # Create a new tenant named my-tenant
-	  kubectl create tenant my-tenant`))
+	  # Create a new tenant named my-tenant, the StorageCluserId will be set to 1 by default
+	  kubectl create tenant my-tenant
+	  
+	  # Create a new tenant named dreamworld with storage cluster set to c1
+	  kubectl create tenant dreamworld --storagecluster=StorageCluserId=c1`))
+
+	defaultStorageClusterId = "1"
```

Comment:
go fmt?

Reply:
I did not know that we support "kubectl create tenant <tenant-name">. Can you add an UT or integration test for this command?

Reply:
When this becomes general supported case, I would suggest using system storage as defaultStorageClusterId.

Reply:
To add protection for failures like this, we need e2e tests. Even integration tests are not enough. If you take a look at the create_namepsace_test.go, you will see those UT test cases are just some toy tests.

Reply:
synced offline. Updated the defaultStorageClusterId to "system". Also created an issue to track the work to move away from hard-coded definition when the default value is defined in API server side. https://github.com/futurewei-cloud/arktos/issues/326